### ISSUE TITLE
DatePicker/Dropdown : HC, Firefox : keyboard focus/hover is not visible(no border treatment)

### ISSUE THREAD

**Issue by bharay** [OfficeDev/office-ui-fabric-react#2229]
### Bug Report
- __Package version(s)__: latest

#### Priorities and help requested (not applicable if asking question):
Are you willing to submit a PR to fix? (No)
Requested priority: (High, Normal) 
Products/sites affected: (VSTS)

#### Describe the issue:
On focus/hover, there is no high contrast border behavior change in firefox for datepicker and dropdown component. This works for IE/edge.

#### Actual behavior:
No border treatment. Checked in https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/datepicker

#### Expected behavior:
As in IE/edge.

### If applicable, please provide a codepen repro:
Can check in https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/datepicker
